### PR TITLE
Save log file to S3 bucket upon restore completion

### PR DESCRIPTION
**Pull request by aleksievelizhanin**

### CODE REVIEW

File: app/backend/src/clj/teet/backup/backup_ion.clj [hunk 1]
```diff
@@ -364,6 +382,7 @@
              prepare-asset-db-for-restore
              restore-tx-file
              migrate
+             log-restore-result
```
Comment (tatut) [inline app/backend/src/clj/teet/backup/backup_ion.clj:385]: what about exception case? I think failure should be logged as well.. so anyone polling for this log file knows that the thing failed

File: app/backend/src/clj/teet/backup/backup_ion.clj [hunk 2]
```diff
@@ -342,6 +342,24 @@
...
+(defn log-restore-result
+  "Write log to bucket into the same file name from which the restore was started"
+  [{{bucket :bucket file-key :file-key} :s3}]
+  (try
+    (let [file (java.io.File/createTempFile file-key ".log")]
+      (log/info "Generating restore log file: " (.getAbsolutePath file))
+      (with-open [w (io/writer file :append true)]
+                 (.write w (str "Backup " file-key " was restored successfully.")))
+      (with-open [in (io/input-stream file)]
+                 (s3/write-file-to-s3
```
Comment (tatut) [inline app/backend/src/clj/teet/backup/backup_ion.clj:354]: indent body of `with-` macros at 2

**Review by tatut — CHANGES_REQUESTED** [review]
log failure

File: app/backend/src/clj/teet/backup/backup_ion.clj (hunk 1 above)
Comment (aleksievelizhanin) [reply app/backend/src/clj/teet/backup/backup_ion.clj:385]: - Exception in saving file can not be logged as something already wrong in saving to S3 bucket.
- The exception in restore thowns Exception. In this case the log-restore-result function will not be called at all. 
- Is there any other case then failure in restore saved in context, for instance, without throwing an exception?

**Review by tatut — APPROVED** [review]

**Pull request merged**